Source/WPEFramework/PluginServer.h

```diff
@@ -1040,6 +1040,11 @@ namespace PluginHost {
                     // Could be that we can now drop the dynamic library...
                     Core::ServiceAdministrator::Instance().FlushLibraries();
                 }
+
```

Comment:
This should be released prior to the FlushLibraries, as potentially the flush might be ignored if the Connection is still holing a reference to anything in that librray. SO please move this to  line 1031